src/agenda/index.js

```diff
@@ -14,12 +14,6 @@ import {AGENDA_CALENDAR_KNOB} from '../testIDs';
 
 const HEADER_HEIGHT = 104;
 const KNOB_HEIGHT = 24;
-//Fallback for react-native-web or when RN version is < 0.44
```

Comment:
This is not a breaking change? How can I test it?

Reply:
this calendar version has dependency on "react-native": "0.62.2" (it is located in devDependencies, it's wrong)
from this version till master branch there is no changes with `View.propTypes.style`